azure-pipelines.yml

```diff
@@ -12,42 +12,41 @@ variables:
   DOCKER_BUILDKIT: 1
 
 stages:
-  - stage: php74
-    displayName: PHP 7.4
+  - stage: php80
```

Comment:
tak 8.0 už taky nikde nemáme, to rovnou posuň na 8.3

Reply:
as you wish, sir :slightly_smiling_face: ne, ciste 8.3 si myslim je jeste too soon (afaik 8.3 nemame jeste nikde, ne?), ale 8.2+ vidim v pohode

Reply:
upravil jsem na PHP 8.2 + 8.3

Reply:
no myslel jsem to tak, že nemá smysl pouštět testy na 8.0, min. requirement bych dal 8.2 jak máš, na 8.3 je určitě too soon na min requirement, ale na testy je to akorát - prostě přesně tak jak jsi to udělal